src/compiler/expression/object/ShorthandPropertyAssignment.ts

```diff
@@ -87,4 +87,9 @@ export class ShorthandPropertyAssignment extends ShorthandPropertyAssignmentBase
 
         return parent.getChildAtIndexIfKindOrThrow(childIndex, SyntaxKind.PropertyAssignment) as PropertyAssignment;
     }
+
+    /**
+     * Removes this property
+     */
+    remove = PropertyAssignment.prototype.remove;
```

Comment:
I'd prefer this to be a method rather than a property declaration so it will work with inheritance if necessary.

Perhaps extract out the remove method found in `PropertyAssignment` to a reusable function (ex. `removeObjectLiteralExpressionMember(this);`) then call it within each method body.